components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useState } from "react";
+import starryEyed from "../images/feedback/starry-eyed-emoji.svg";
+import happy from "../images/feedback/happy-emoji.svg";
+import meh from "../images/feedback/meh-emoji.svg";
+import crying from "../images/feedback/crying-emoji.svg";
+import { trackEvent } from "../Analytics";
+
+function FeedbackComponent(props: { onClose: () => void; onSubmit: () => void; isModal: boolean }) {
+    const [text, setText] = useState<string>("");
+    const [selectedEmoji, setSelectedEmoji] = useState<number | undefined>();
+
+    const height = props.isModal ? "300px" : "";
+
+    const onSubmit = () => {
+        if (selectedEmoji) {
+            const feedbackObj = {
+                score: selectedEmoji,
+                feedback: text,
+                href: window.location.href,
+                path: window.location.pathname,
+            };
+            trackEvent("feedback_submitted", feedbackObj);
+        }
+
+        props.onSubmit();
+    };
+
+    const handleClick = (emojiScore: number) => {
+        setSelectedEmoji(emojiScore);
+    };
+
+    const emojiGroup = (width: number) => {
+        return (
+            <>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === 4 ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(4)}
+                >
+                    <img src={starryEyed} alt="starry eyed emoji" width={width || "24px"} title="4" />
```

Comment:
Is there are a reason why this `<img>` tag has different attributes to the others? If it were the same it would be easier to remove the duplication in this component.